config/environments/development.rb

```diff
@@ -50,6 +50,10 @@
   # Suppress logger output for asset requests.
   config.assets.quiet = true
 
+  config.hosts += [
+    "publisher.dev.gov.uk",
+  ]
```

Comment:
```suggestion
  config.hosts << "publisher.dev.gov.uk"
```

This is slightly smaller, but also I think the convention is actually just to do `config.hosts.clear`, as per https://github.com/alphagov/content-publisher/blob/69bed3535de1e65bda9d5909a3bc56a319e46576/config/environments/development.rb#L16 and https://github.com/alphagov/email-alert-api/blob/ab738b54c891b264ace58b91463cdecfa78d47b5/config/environments/development.rb#L56

Reply:
Thanks Tom - I'll change it 👍